examples/vault-iam-auth/README.md

```diff
@@ -100,20 +113,91 @@ vault write \
 See the whole example script at [user-data-vault.sh][user_data_vault].
 
 
-### Authenticating from an instance
+### Authenticating from a client
+
+#### With an HTTP request
+
+The [vault-consul-ami][vault_consul_ami] includes a [python script][py_sign] called
+`sign-request.py`. We use python here instead of bash so we take advantage of the
+`boto3` AWS SDK library. This script is a modified version of the Python 2.x example
```

Comment:
(the same pattern should work with the AWS SDK in any other supported language such as Go, Java, Ruby, ...)